IssueComment
  { issueCommentUpdatedAt = 2013 (-08) (-30) 17 : 39 : 30 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 1461947
        , simpleUserLogin = N "neumino"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/1461947?v=3"
        , simpleUserUrl = "https://api.github.com/users/neumino"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/comments/23577630"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/rethinkdb/issues/1387#issuecomment-23577630"
  , issueCommentCreatedAt = 2013 (-08) (-30) 17 : 39 : 30 UTC
  , issueCommentBody =
      "@vivekp the native python implementation of protobuf is slow, especially when you deal with big documents.\r\nI guess you are running the clients on the same machine as the server right? If that's the case, I wouldn't be surprise that this is the problem.\r\n\r\nThe error you were getting was a bug. It's fixed -- See https://github.com/rethinkdb/rethinkdb/issues/1359\r\nRunning `sudo pip install rethinkdb --upgrade` should install the driver 1.8.0-1 which can work with the C++ backend.\r\n\r\nThis blog post has some numbers between the different backends: http://www.rethinkdb.com/blog/1.7-release/\r\n "
  , issueCommentId = 23577630
  }